sql/mob_spell_lists.sql

```diff
@@ -4929,7 +4929,6 @@ INSERT INTO `mob_spell_lists` VALUES ('Hakutaku',486,249,1,255); -- blaze_spikes
 -- Shen (487)
 INSERT INTO `mob_spell_lists` VALUES ('Shen',487,172,1,255); -- water_iv (1~255)
 INSERT INTO `mob_spell_lists` VALUES ('Shen',487,201,1,255); -- waterga_iii (1~255)
-INSERT INTO `mob_spell_lists` VALUES ('Shen',487,214,1,255); -- flood (1~255)
```

Comment:
I thought Shen did cast Flood

https://www.youtube.com/watch?v=L76e2twvwz4 @3:14 Minutes shows starts casting from what I found

https://ffxiclopedia.fandom.com/wiki/Shen also shows cast it

Reply:
In my research watching videos and getting a capture of Shen, he only casts Flood to summon his water elementals. If you see my Shen code, he instant casts Flood so that he can spawn a pet at the location of the player that he cast flood on.

Reply:
Ah ok. I see. No worries, was just asking since was recently added and then removed.